### PR TITLE
editorial changes

### PR DESCRIPTION
**Pull request by krishahn**
There's a list of links at the bottom of the page that don't show up in the published version. Do you want them to appear?

### CODE REVIEW

File: docs/configuring-api-console.md [hunk 1]
```diff
@@ -18,37 +18,37 @@ console.narrow = true;
...
+| `proxy` | Sets the proxy URL for the HTTP requests sent from the console. To alter all URLs before sending the data to a transport library, set the attribute and prefix the URL with its value. | `String`
+| `proxy-encodeUrl` | If required by the `proxy` the URL is URL-encoded. | `Boolean` |
+| `no-try-it` | Disables the "try it" button in the method documentation view. The request editor and the response viewer is still available, but you must open it programmatically by setting `page` property to ` request`. | `Boolean` |  
+| `manual-navigation` | Disables navigation in the drawer and renders the navigation full screen when requested. Use in the narrow layouts with the `narrow` property. Set the `navigationOpened` property to `true` or `false` to open/close the navigation. | `Boolean` |
+| `navigation-opened` | If this and the `manual-navigation` attribute are set, then the full screen navigation will open/close. | `Boolean` |
```
Comment (jarrodek) [inline docs/configuring-api-console.md]: I don't get it. 

> If this and the `manual-navigation` attribute are set,

It should be: `If set` - as when the attribute is set, and `the manual-navigation attribute is set`. There's only one attribute.

**Comment by sichvoge** [comment]
Links at the bottom need to stay. @krishahn please see Pawel's comments.

**Comment by jarrodek** [comment]
The only link that is not used is this one: `[camel case]: https://en.wikipedia.org/wiki/Camel_case` and can be removed. Other links are used in the markup.

**Comment by jarrodek** [comment]
LGTM

**Pull request merged**